sqlglot/tokens.py

```diff
@@ -1246,3 +1246,45 @@ def _extract_string(self, delimiter: str, escapes=None) -> str:
                 text += self.sql[current : self._current - 1]
 
         return text
+
+    @classmethod
+    def tokenize_native(cls, sql: str) -> t.List[Token]:
+        from sqlglotrs import (  # type: ignore
+            Tokenizer as RsTokenizer,
+            TokenizerSettings as RsTokenizerSettings,
+        )
+
+        settings = RsTokenizerSettings(
```

Comment:
doesn't this mean this will happen every time we call tokenize_native? can we do this on module load instead?